docs/contributing.rst

```diff
@@ -83,6 +83,16 @@ Running Tests
 Linting
 -------
 
+This project uses [pre-commit](https://pre-commit.com/) for linting.
```

Comment:
This is markdown syntax and does not get rendered. :)
Please stay with rst.

Reply:
my bad! I live in a world of markdown usually.

I *think* i've fixed this properly, but please give this a sanity-check for me!